dropshot_endpoint/src/lib.rs

```diff
@@ -53,6 +54,15 @@ struct Metadata {
 
 const DROPSHOT: &str = "dropshot";
 
+const ERR_MSG_USAGE: &str = "Try using the following endpoint format:
```

Comment:
The wording here strikes me as overly coy. How about: "Endpoint handers must have the following signature"